stylesheets/stylesheet.css

```diff
@@ -522,6 +522,11 @@ form {
   word-break: break-all;
 }
 
+.projects .contribute {
+  font-size: 14px;
+  word-break: break-all;
```

Comment:
There's some weird trimming here:

<img width="765" src="https://cloud.githubusercontent.com/assets/359239/25067296/27e09246-2282-11e7-89f3-729d001350e5.png">

Should this be `keep-all`?